wchat/screens/shared/components/ContactTable/index.js

```diff
@@ -12,6 +12,7 @@ export default class ContactTable extends Component {
       id={item.id}
       title={item.username}
       avatarUrl={item.avatar}
+      onPress={()=>{this.props.onSelect(item.id)}}
```

Comment:
declare onPress like a function of the class. if you do a arrow function into the render, on each render call, will be create a new function. So movin it to a function of the class, avoid this problem.